AutoNode/initialize.py

```diff
@@ -0,0 +1,213 @@
+                raise RuntimeError("Bad BLS import, missing BLS passphrase file.")
+        return None
+    if bls_keys:
+        return getpass.getpass(f"Enter passphrase for all {Typgpy.UNDERLINE}{len(bls_keys)} "
+                               f"imported{Typgpy.ENDC} BLS keys\n> ")
+    return getpass.getpass(f"Enter passphrase for {Typgpy.UNDERLINE}generated{Typgpy.ENDC} BLS key\n> ")
+
+
+def _import_wallet_passphrase():
+    wallet_pass = [x for x in os.listdir(imported_wallet_pass_file_dir) if not x.startswith('.') and x.endswith(".pass")]
+    for p in wallet_pass:
+        tok = p.split('.')
+        if len(tok) != 2 or not tok[0].startswith("one1"):
+            print(f"{Typgpy.FAIL}Imported wallet passphrase file {p} has an invalid file format. "
+                  f"Must be `<ONE-address>.pass`{Typgpy.ENDC}")
+            raise RuntimeError("Bad wallet passphrase import")
+        if validator_config['validator-addr'] == tok[0]:
+            with open(f"{imported_wallet_pass_file_dir}/{p}", 'r', encoding='utf8') as f:
+                return f.read().strip()
+    return getpass.getpass(f"Enter wallet passphrase for {validator_config['validator-addr']}\n> ")
+
+
+def _import_bls(passphrase):
+    """
+    Import BLS keys using imported passphrase files if passphrase is None.
+    Otherwise, use passphrase for imported BLS key files or generated BLS keys.
+
+    Assumes that imported BLS key files and passphrase have been validated.
+    """
+    bls_keys = [x for x in os.listdir(bls_key_dir) if not x.startswith('.') and x.endswith(".key")]
+    bls_pass = [x for x in os.listdir(imported_bls_pass_file_dir) if not x.startswith('.') and x.endswith(".pass")]
+    if passphrase is None:
+        for k in bls_pass:
+            shutil.copy(f"{imported_bls_pass_file_dir}/{k}", bls_key_dir)
+            subprocess.call(f"chmod go-rwx {imported_bls_pass_file_dir}/{k}", shell=True, env=os.environ)
+        for k in bls_keys:  # Verify Passphrase
+            try:
+                cli.single_call(f"hmy keys recover-bls-key {bls_key_dir}/{k} "
+                                f"--passphrase-file {imported_bls_pass_file_dir}/{k.replace('.key', '.pass')}")
+            except RuntimeError as e:
+                print(f"{Typgpy.FAIL}Passphrase file for {k} is not correct. Error: {e}{Typgpy.ENDC}")
+                raise RuntimeError("Bad BLS import") from e
+        return [k.replace('.key', '').replace('0x', '') for k in bls_keys]
+
+    with open("/tmp/.bls_pass", 'w', encoding='utf8') as fw:
+        fw.write(passphrase)
+        subprocess.call(f"chmod go-rwx /tmp/.bls_pass", shell=True, env=os.environ)
+    if len(bls_keys) > 0:
+        if node_config['shard'] is not None:
+            print(f"{Typgpy.WARNING}[!] Shard option ignored since BLS keys were imported.{Typgpy.ENDC}")
+            time.sleep(3)  # Sleep so user can read message
+        for k in bls_keys:
+            try:
```

Comment:
maybe invert the for/try

Reply:
want to print the key that caused the error